Functions/xASL_init_LoadMetadata.m

```diff
@@ -170,10 +170,20 @@
 %xASL_bids_LoadParticipantTSV 
 
     %% Load participants.tsv
-    PathTSV = fullfile(x.D.ROOT, 'Participants.tsv');
+    PathTSV = fullfile(x.D.ROOT, 'participants.tsv');
+    PathTSVold = fullfile(x.D.ROOT, 'Participants.tsv');
+    bParticipantsTSV = false;

```

Comment:
to me, it is better to rename it if it detects the old one...

Reply:
You're right. That's probably the more straight forward way to do this. 
2b189c8 👍

Reply:
thanks for the edit. Same as above - do we need the temp? Should we check for existence of both